formatter/friendly.go

```diff
@@ -125,17 +125,19 @@ func (f *Friendly) printStatistics(w io.Writer, header string, stats map[string]
 		formatted = append(formatted, []string{color.GreenString(fmt.Sprintf("%d", entry.failures)), entry.name})
 	}
 	fmt.Fprintln(w, header)
-	fmt.Fprintln(w, f.table(formatted))
+	fmt.Fprintln(w, table(formatted))
 }
 
-func (*Friendly) table(rows [][]string) string {
-	buf := new(bytes.Buffer)
-	table := tablewriter.NewWriter(buf)
-	table.SetBorder(false)
-	table.SetColumnSeparator("")
-	table.SetRowSeparator("")
-	table.SetAutoWrapText(false)
-	table.AppendBulk(rows)
-	table.Render()
+func table(rows [][]string) string {
+	var buf bytes.Buffer
+	tw := tabwriter.NewWriter(&buf, 0, 0, 2, ' ', 0)
+	for _, row := range rows {
+		tw.Write([]byte{'\t'})
```

Comment:
I don't get why lines start with tab

https://go.dev/play/p/tu8LLRA5LCZ

```
---|---a|---b|---aligned|
```

Reply:
To build this indentation:

<img width="1027" alt="Screenshot 2025-05-17 at 10 52 45" src="https://github.com/user-attachments/assets/73b64a19-6ab9-4fea-a838-70b7553a7075" />

Reply:
As long as we are fine that result are left aligned like this

https://go.dev/play/p/7UT-dXUTa5K

![Screenshot_20250517_134834_Firefox.jpg](https://github.com/user-attachments/assets/c9a0c8fb-0f3b-4e13-940d-6a95d3243057)

Note: I used debug mode and dash padding to debug how data are displayed. We won't use this. It's for helping understanding.

So now, I'm fine if everyone agree it's fine that words are left aligned

I have no idea how code was formatted before with the tabwritter lib we had.

From what I saw, there is nothing in std tawritter lib to align the first column to right, and second to left.